build/ci/code_scanning/run_csi_code_scan.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -x
+
+CODE_SCANNING_STAGE=$1
+OUTPUT_PATH="`pwd`/build/reports"
+
+if [ ${CODE_SCANNING_STAGE} == "operator" ]
+then
+  docker build -f build/ci/code_scanning/Dockerfile-csi-operator-code-scan -t csi-operator-code-scan . && \
```

Comment:
if/else can be avoided by requiring the exact name, e.g. "docker build -f build/ci/code_scanning/Dockerfile-{TARGET-NAME} -t {TARGET-NAME}"